front/tests/assets/operationStudies/timesAndStops/expectedOutputsCellsData.json

```diff
@@ -25,8 +25,8 @@
     "margin": {
       "theoretical": "1 min/100km",
       "theoreticalS": "9 s",
-      "actual": "167 s",
-      "difference": "159 s"
+      "actual": "114 s",
```

Comment:
Understood that one: closed-signal reception leads to a slower base speed-profile (so longer base duration to reach the next stop) and the actual margin is the diff between that base duration and the duration of the actual speed-profile used to target both margin and provided arrival-time.
So that diff is lowered ✅